examples/audio/uk_ireland_accent_recognition.py

```diff
@@ -0,0 +1,664 @@
+"""
+Title: UK & Ireland Accent Recognition Using Transfer Learning
+Author: [Fadi Badine](https://twitter.com/fadibadine)
+Date created: 16/04/2022
+Last modified: 16/04/2022
+Description: This notebook trains a model to classify UK & Ireland accents using feature extraction from Yamnet.
+"""
+
+"""
+# Prerequisits
```

Comment:
- All section titles should use `##` not `#`
- Please add an introduction section explaining what the example is about, what the reader will learn, what dataset you use, etc.